orttraining/orttraining/core/framework/gradient_graph_builder.cc

```diff
@@ -169,6 +169,20 @@ NodeSet GradientGraphBuilder::ReverseBFSWithStopGradient(const NodeSet& nodes) c
                             << " of node: " << n->Name();
```

Comment:
Line 166:          const std::unordered_set<size_t>* edges = GetStopGradientEdges(*n);
Can you move it out of the loop, will save execution cycles.